js/models/wizard_law_enforcement_records.js

```diff
@@ -0,0 +1,72 @@
+import {
+  answer, question, summary, yesNoQuestion,
+} from '../util/wizard_helpers';
+
+const question22 = () => yesNoQuestion('q22', {
+  yes: summary('m54'),
+  no: summary('m55'),
+});
+
+/**
+ * @type {WizardQuestion}
+ */
+const lawEnforcementRecordsJourney = question('q20', [
+  answer('literal:State or local', summary('m52')),
+  answer(
+    'literal:Federal law enforcement',
+    question('q21', [
+      answer('literal:Law enforcement policies and programs', summary('m53')),
+      answer('literal:Records about a specific criminal or civil case', yesNoQuestion('q22', {
+        yes: summary('m54'),
+        no: summary('m55'),
+      })),
+      answer('literal:Federal prison and parole records', summary('m56')),
+      answer('literal:Background check records', summary('m34')),
+      answer('literal:Court records', summary('m57')),
+      answer('literal:Oversight of the environment and agriculture', summary('m58')),
+      answer('literal:Oversight of U.S. financial institutions', summary('m59')),
+      answer('literal:Oversight of the workplace', summary('m60')),
+      answer('literal:Consumer protection', summary('m61')),
+      answer('literal:Oversight of federal agencies', summary('m62')),
+    ]),
+  ),
+  answer("literal:I don't know", question('q23', [
+    answer('literal:911 calls', summary('m52')),
+    answer('literal:Arrest and/or investigation records', question('q24', [
+      answer('literal:Federal investigation'),
```

Comment:
When I get to here in the journey, I get an `Uncaught TypeError: Cannot read properties of undefined (reading 'type')` at `at eval (wizard_store.js:203:23)`, because there's no `answer.next`.  Looks like it should route to q22?

Reply:
Ooo, good catch.